Automatically try to obtain auth for portalConfig if not present

When updating the config manually to...
```
defaultPortal: 'DEV'
# List of portals that are intended to be used
portals:
  # Portal set up to use an API Key
  - name: 'DEV'
    portalId: <portalId>
    authType: 'personalaccesskey'
    personalAccessKey: 'xxxxxxx'
    env: 'QA'
```

an error is encountered:
```[ERROR] The portalId <portalId> is missing authentication configuration```

We should try to obtain the missing auth data in the config if we have the means to before throwing an error.

--- a/packages/cms-lib/lib/config.js
+++ b/packages/cms-lib/lib/config.js
@@ -219,7 +219,7 @@ const updatePortalConfig = configOptions => {
       tokenInfo,
     };
   }
-  const env = getConfigEnv(environment);
+  const env = getConfigEnv(environment || portalConfig.env);
   const mode = defaultMode && defaultMode.toLowerCase();
   const nextPortalConfig = {
     ...portalConfig,

